README.md

```diff
@@ -14,8 +14,8 @@ evm.codes is brought to you by [smlXL](https://www.smlxl.io), powered by open-so
 
 The app requires the following dependencies:
 
-- [NodeJS](https://nodejs.org/) >= 18
-- [Yarn](https://yarnpkg.com/)
+- [NodeJS](https://nodejs.org/) >= 20
```

Comment:
Node 20 is the active LTS version currently and it aligns this project with other smlxl projects.